ISISComputingGroup/ibex_gui
# Ticket 4786: Stop OPIs moving focus

### Ticket

See https://github.com/ISISComputingGroup/IBEX/issues/4786.

### To test

1. Run `install_pre_commit_hook.bat`
2. Try and do a commit
3. Confirm that the commit fails
4. Confirm that the hook correctly adds dummy buttons to all OPIs
5. Rerun the commit and confirm the hook passes as OPIs have dummy button

---

#### Code Review


### Final Steps
